src/tools/annotation/CircleRoiTool.js

```diff
@@ -43,6 +43,9 @@ export default class CircleRoiTool extends BaseAnnotationTool {
       name: 'CircleRoi',
       supportedInteractionTypes: ['Mouse', 'Touch'],
       svgCursor: circleRoiCursor,
+      configuration: {
+        drawHandles: true,
+      },
```

Comment:
This is a tricky one. We may want to adopt the `drawHandlesOnHover` approach here. Example:

- `drawHandles` usually works be showing the handle if the _handle_ is hovered
- `drawHandlesOnHover` usually works by showing the handles if `pointNearTool` is true

For a circle, there's no clear "line end" like in a `length` tool. You'd have to trace the entire circle to hunt for where the handle might be.

Reply:
I think the previous OHIF Meteor version simply ignored ROI tools like this. It continued to show their handles instead of addressing the UX issue here. Only tools, like length, had their handles hidden.

For your current task, you can continue without supporting tools like this, or you can work in support. Whichever you have the appetite for is fine by me.

Reply:
Removed. Thanks.